src/helpers.js

```diff
@@ -337,12 +337,27 @@ define(function (require) {
 
     return bufferSize;
   }
+ var safeBins =  p5.prototype.safeBins  = function(bins) {
```

Comment:
do we need to add this to the p5 prototype, or can this be a private method that is only used internally?

If it's only added to the prototype for the purpose of writing a test, I think the test should instead be on the public methods that utilize this function behind the scenes.

Reply:
@therewasaguy  the main problem i face was using AMD here , I was unable to use safeBins defined in src/helper.js in test/tests/helpers.test.js , so I decided to put this method in prototype object of p5 ,
and I am realising now , that it is not the good way of doing it !,
so  I will add tests on   public function that will wrap this function ! and remove it from p5.prototype